sap/5.3/modules/ROOT/pages/sap-connector-reference.adoc

```diff
@@ -18,6 +18,16 @@ Workarounds:
 
 * SAP Connector does not support Bitronix Transaction Manager. 
 
+* The reconnection strategy on SAP (applies to operations only) is improved. With this enhancement, when a connection fails on the FunctionSource or DocumentSource, the JCo server automatically tries to reestablish a connection in the following way:
+
+ ** If a connection issue occurs, the JCo server retries immediately.
+
+ ** If the connection is not restored immediately, the JCo server waits for one second and retries.
```

Comment:
please mention that this relies exlusively on the jco mechanism, the mule mechanism has no effect here.

Reply:
exactly.

" The reconnection strategy on SAP (applies to operations only) is improved" I'm not sure that that means

Reply:
Fixed.